charts/stable/sonarr/values.yaml

```diff
@@ -36,5 +42,12 @@ persistence:
     enabled: true
     mountPath: "/config"
 
+# might be supported on sonarr v4 upon released
+postgresql:
+  enabled: false
```

Comment:
With this disabled.. it does not have a `.Values.postgresql.url.plain` value to trim...

Reply:
Ok, so I'll remove postgres since sonarr v3 doesnt support it.